src/Hangman/Exceptions/all-exceptions.filter.ts

```diff
@@ -0,0 +1,48 @@
+import { Catch, ArgumentsHost, HttpStatus, Logger } from '@nestjs/common';
```

Comment:
from http://draganatanasov.com/2019/06/24/handle-your-exceptions-by-using-global-exception-filter-in-a-nestjs-application/